MonitorizareVotTests/APITests.swift

```diff
@@ -83,7 +83,7 @@ class APITests: XCTestCase {
         let knownPin = correctPin
         let waiter = expectation(description: "Forms")
         sut?.login(withPhone: knownPhone, pin: knownPin, then: { error in
-            self.sut?.fetchForms(inSet: 1, then: { (forms, error) in
+            self.sut?.fetchForm(withId: 1, then: { (forms, error) in
```

Comment:
Fixed unit test to enable compiling. However they do fail and given they call the network, they could be very flaky at best. I would comment them out and maybe look at refactoring the service in a way that would enable unit testing without network being reached, if unit tests become more frequent and a pipeline to run them is created

Reply:
These tests were just there for the refactoring process, where I had to work with 2 APIs in parallel, they're not meant to be used as unit tests for the iOS project. I'll remove them at some point. Also they fail because changes were made to the API since they were written. Thanks for the reminder!